packages/components/icon/package.json

```diff
@@ -33,7 +33,7 @@
   "dependencies": {
     "@contentful/f36-core": "^5.1.0",
     "@contentful/f36-tokens": "^5.1.0",
-    "@phosphor-icons/react": "^2.1.5",
+    "@phosphor-icons/react": "^2.1.8",
```

Comment:
```suggestion
    "@phosphor-icons/react": "2.1.8",
```

Reply:
If I can get npm to share versions properly it shouldn't be necessary to pin it everywhere. I guess we can, though 🤔